app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
                     WHERE p4:i.id = p.id AND mut2:i.id :operator :value
                 )'
             ),
+            'review_status' => new DoctrineSwitchFilterMapping
+            (
+                [
+                    Presentation::ReviewStatusNoSubmitted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusNoSubmitted,
+                        "selp IS NULL OR (p.status <> 'Received' AND p.status <> 'Accepted')"
```

Comment:
please do https://github.com/OpenStackweb/summit-api/blob/2d46fc910d303115fc680f5dc9763acbf68728bf/app/Repositories/Summit/DoctrineSummitEventRepository.php#L307
p.status is null

Reply:
done